homeWork7/task1.js

```diff
@@ -0,0 +1,12 @@
+const numbers = [2, -5, 0, 7, -3, 0, 10, -8]
+let positiveCount = 0
+let negativeCount = 0
+let zeroCount = 0
+for (let i of numbers ){
```

Comment:
use const and more meaningful name
```js
for (const number of numbers )
```

because `i` usually interpreted as index